packages/lesswrong/server/tableOfContents.ts

```diff
@@ -206,7 +207,10 @@ async function getTocAnswers (document: DbPost) {
   if (forumTypeSetting.get() === 'AlignmentForum') {
     answersTerms.af = true
   }
-  const answers = await Comments.find(answersTerms, {sort:questionAnswersSort}).fetch()
+  const location = useLocation();
+  const { query } = location;
+  const sortOrder = query?.answersSorting || "top";
+  const answers = await Comments.find(answersTerms, {sort:questionAnswersSortings[sortOrder]}).fetch()
```

Comment:
I haven't been able to get the table of contents to show up on the local server, so probably this doesn't work

Reply:
Normal posts only show ToCs when the post in question has headers — enough structure to actually deserve a ToC.

It seems like a questionable (😉) choice to me, but even question posts only show ToCs when the post contents have headers. So maybe you weren't testing with headers. However in my testing on your branch I couldn't get questions with headers to render ToCs. (Whereas on master it rendered.) So I'm guessing there's a bug. Maybe it's the thing the lint is complaining about?

Reply:
I think this is basically an oversight that should be fixed, although probably would handle it in a separate PR

Reply:
You mean adding the ToC to questions with answers even if they don't have 3 headings?
Happy to make a PR, it might be just:

```diff
 const tableOfContents = extractTableOfContents(html)
 let tocSections = tableOfContents?.sections || []
  
- if (tocSections.length >= MIN_HEADINGS_FOR_TOC) {
-   const tocAnswers = await getTocAnswers(document)
+ tocSections.push(...await getTocAnswers(document))
+ if (tocSections.length >= MIN_SECTIONS_FOR_TOC) {
```